package.json

```diff
@@ -55,12 +57,14 @@
     "storybook": "^8.3.3",
     "stylelint": "^15.10.0",
     "stylelint-config-standard": "^31.0.0",
+    "svelte-check": "^4.1.0",
     "svelte-loader": "^3.2.3",
     "tailwindcss": "^3.3.0",
     "typescript-eslint": "^8.8.0",
     "vite": "^4.4.9"
   },
   "dependencies": {
+    "@melt-ui/svelte": "^0.86.2",
```

Comment:
Looks like this is only used in one place as a type? Feel like we could remove it, or because it is only a type at least make it a devDep?

https://github.com/frequency-chain/style-guide/blob/fc31203adcc2855f9e5bec16d95a4bd06b918833/src/lib/shadcnComponents/ui/select/select-trigger.svelte.d.ts#L27